soundbay/models.py

```diff
@@ -364,3 +364,26 @@ def __init__(self, num_classes=2, pretrained=True):
     def forward(self, x):
         x = x.repeat(1, 3, 1, 1)
         return self.resnet(x)
+
+class EfficientNet2D(nn.Module):
+    """
+    EfficientNet model for 3 channel ("RGB") input.
+    """
+    def __init__(self, num_classes=2, pretrained=True):
+        super(EfficientNet2D, self).__init__()
+        
+        # Load a pre-trained EfficientNet-B7 model from torchvision
+        self.efficientnet = models.efficientnet_b7(pretrained=pretrained)
```

Comment:
maybe we should have the chosen size `b7` as a parameter, coming from the configuration - I think that even `b0` worth trying and might be enough